pkgs/top-level/dotnet-packages.nix

```diff
@@ -220,6 +220,37 @@ let self = dotnetPackages // overrides; dotnetPackages = with self; {
 
   # SOURCE PACKAGES
 
+  Boogie = buildDotnetPackage rec {
+    baseName = "Boogie";
```

Comment:
Oh, and similarly the commit message would be "dotnetPackages.Boogie: init at 2017-01-03"…